server/http.go

```diff
@@ -101,6 +102,30 @@ func (p *Plugin) submitFormPMIForMeeting(w http.ResponseWriter, r *http.Request)
 		Text: fmt.Sprintf("You have selected `%s` to start the meeting.", action),
 	}
 
+	userPMISettingPref, err := p.getPMISettingData(userID)
+	if err != nil {
+		http.Error(w, err.Error(), http.StatusInternalServerError)
+		return
+	}
+
+	if userPMISettingPref == "" {
+		val := falseString
+		meetingIDType := "unique"
+		if action == usePersonalMeetingID {
+			val = trueString
+			meetingIDType = "personal"
+		}
```

Comment:
The previous default (and only) preference was hardcoded "personal meeting id", so we're changing the default behavior that the user had before this release if the default fallback is "unique". Probably doesn't matter though since we are overwriting it right afterwards